cuqi/experimental/mcmc/_sampler.py

```diff
@@ -45,8 +72,10 @@ def __init__(self, target: cuqi.density.Density, initial_point=None, callback=No
         # Choose initial point if not given
         if initial_point is None:
             initial_point = np.ones(self.dim)
+
+        self.initial_point = initial_point
         
-        self._samples = [initial_point]
+        self._samples = [initial_point] # Remove?
```

Comment:
Do we need to capture this in one of the issues we have or create a new issue (Consider whether to include initial_point in the initial state of self._samples?)

Reply:
Done